src-vue/src/plugins/store.ts

```diff
@@ -28,11 +28,11 @@ export const store = createStore({
         return {
             current_tab: Tabs.PLAY,
             developer_mode: false,
-            game_path: "this/is/the/game/path",
+            game_path: undefined,
```

Comment:
Since `game_path` is a string, shouldn't its default value be `""`?

Reply:
I tested `undefined` and it worked in terms of rejecting an install. The issue with `""` that I'd have to add extra logic to back-end to classify it as a bad install path. Otherwise it would just install to `./` ^^